modules/hosted-control-planes-metrics-sets.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-metrics-sets_{context}"]
+= Configuring metrics sets for hosted control planes
+
+Hosted control planes for Red Hat {product-title} creates `ServiceMonitor` resources in each control plane namespace that allow a Prometheus stack to gather metrics from the control planes. The `ServiceMonitor` resources use metrics relabelings to define which metrics are included or excluded from a particular component, such as etcd or the Kubernetes API server. The number of metrics that are produced by control planes directly impacts the resource requirements of the monitoring stack that gathers them. 
```

Comment:
Is "creates" right here vs. "create"? Wasn't sure if this was meant to be written like a feature or if this was accidental. Ignore if "creates" is accurate.

Reply:
Yep - "creates" is accurate. The feature name is "hosted control planes for Red Hat OpenShift Container Platform".